Source/WebCore/Modules/mediastream/UserMediaController.h

```diff
@@ -68,41 +68,48 @@ class UserMediaController : public Supplement<Page> {
     static UserMediaController* from(Page* page) { return static_cast<UserMediaController*>(Supplement<Page>::from(page, supplementName())); }
 
 private:
-    UserMediaClient* m_client;
+    const WeakPtr<UserMediaClient> m_client;
 
     WeakHashSet<Document, WeakPtrImplWithEventTargetData> m_voiceActivityDocuments;
     bool m_shouldListenToVoiceActivity { false };
 };
 
 inline void UserMediaController::requestUserMediaAccess(UserMediaRequest& request)
 {
-    m_client->requestUserMediaAccess(request);
+    if (RefPtr mediaClient = m_client.get())
```

Comment:
```suggested
    if (RefPtr client = protectedClient())
```

Reply:
Actually, our style is:
```
if (RefPtr client = m_client.get())
```
We never use `protectedFoo()` functions to initialize local variables since the name is longer and there is no perf /safety benefit to be. Please do not apply this review feedback.